src/scripting/exposed_object.hpp

```diff
@@ -0,0 +1,70 @@
+//  SuperTux -- ExposedObject class
+//  Copyright (C) 2016 Tobias Markus <tobbi.bugs@googlemail.com>
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#ifndef HEADER_SUPERTUX_SCRIPTING_EXPOSED_OBJECT_HPP
+#define HEADER_SUPERTUX_SCRIPTING_EXPOSED_OBJECT_HPP
+
+#include "scripting/scripting.hpp"
+#include "scripting/squirrel_util.hpp"
+#include "supertux/script_interface.hpp"
+#include "util/log.hpp"
+
+/**
+ * @class ExposedObject
+ * @param class S: GameObject class (e.g. Gradient)
+ * @param class T: Scripting class (e.g. scripting::Gradient)
+ */
```

Comment:
Are you sure doxygen takes comments like this one? It seems to be JSDoc style. I don't see this in <http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html>.